fabric-structure-api-v1/build.gradle

```diff
@@ -1,2 +1,10 @@
 archivesBaseName = "fabric-structure-api-v1"
 version = getSubprojectVersion(project, "1.1.4")
```

Comment:
Should I update this???????

Reply:
No, it generally gets handled after it’s been merged.